### PR TITLE
Use neutrino in integration tests in stead of RPC.

### LINKED ISSUE #518: Use Neutrino in integration tests instead of RPC

### PR DESCRIPTION
**Pull request by valentinewallace**
Closes #518.

### CODE REVIEW

**Comment by erkarl** [comment]
Running the above changes results in the integration tests hanging after `should get public key node1`.

screen shot 2018-08-21 at 11 06 18

File: public/lnd-child-process.js [hunk 1]
```diff
@@ -57,27 +57,38 @@ module.exports.startLndProcess = async function({
...
+    ]);
+  }
+  // set default production settings if no custom flags
+  if (!isDev && !lndArgs.length) {
+    args = args.concat([
+      '--bitcoin.node=neutrino',
+      '--neutrino.connect=btcd0.lightning.engineering',
+      '--neutrino.connect=127.0.0.1:18333',
+    ]);
+  }
```
Comment (tanx) [inline public/lnd-child-process.js]: @valentinewallace I refactored the settings to always use Neutrino for integration tests as well as running the app in dev mode. Let me know what you think

Comment (valentinewallace) [reply public/lnd-child-process.js]: I think that makes sense. That occurred to me as well, but I wasn't sure what we thought our users running in dev mode would want. LGTM, i'll merge! :)

**Comment by erkarl** [comment]
Interesting, when clearing the `data/lnd` folders the integration tests started passing on this branch, too. LGTM 👍

**Pull request merged**